n_serial.c

```diff
@@ -76,7 +76,7 @@ const char *serialNoteTransaction(char *request, char **response)
         uint32_t jsonbufAvailLen = (jsonbufAllocLen - jsonbufLen);
 
         // Append into the json buffer
-        const char *err = serialChunkedReceive((uint8_t *)(jsonbuf + jsonbufLen), &jsonbufAvailLen, true, (NOTECARD_TRANSACTION_TIMEOUT_SEC * 1000), &available);
+        const char *err = serialChunkedReceive((uint8_t *)(jsonbuf + jsonbufLen), &jsonbufAvailLen, true, (NOTECARD_INTRA_TRANSACTION_TIMEOUT_SEC * 1000), &available);

```

Comment:
Should this be NOTECARD_INTER_TRANSACTION_TIMEOUT_SEC? It was 30 before. Now it's 1.

Reply:
The `NOTECARD_INTER_TRANSACTION_TIMEOUT` is satisfied above, then it comes down here and this should actually be the `NOTECARD_INTRA_TRANSACTION_TIMEOUT`.